packages/core/build/config.js

```diff
@@ -135,27 +136,64 @@ const copyConfig = base => {
 
 const generateSWConfig = is_release => ({
     cleanupOutdatedCaches: true,
-    exclude: [
-        /CNAME$/,
-        /index\.html$/,
-        /404\.html$/,
-        /^localstorage-sync\.html$/,
-        /\.map$/,
-        /sitemap\.xml$/,
-        /robots\.txt$/,
-        /manifest\.json$/,
-        /^public\/images\/favicons\//,
-        /^favicon\.ico$/,
-        /^apple-app-site-association/,
-        /^assetlinks.json/,
-        /^.well-known\//,
-        /^account\//,
-        /^js\/smartcharts\//,
-        /^bot\//,
-        /^media\//,
-        /^trader\//,
-        /^cashier\//,
-        /^js\/core\.[a-z_]*-json\./,
+    exclude: [/\**/],
+    runtimeCaching: [
+        {
+            urlPattern: /public\/(images|sprites)\/(?!.*favicons).*$/,
+            handler: 'CacheFirst',
+            options: {
+                cacheName: 'assets',
+                expiration: {
+                    maxAgeSeconds: 60 * 60 * 24,
+                },
+            },
+        },
+        {
+            urlPattern: ({ url }) => {
+                return url.pathname.match(/^\/js\/(?!(.*((core\.[a-z_]*-json\.)|smartcharts))).*$/);
```

Comment:
shouldn't it be `js` instead of `json`?

Reply:
also, can we remove smartcharts from here as we added it at line 164?

Reply:
@balakrishna-binary here smartcharts is needed as this pattern will cache everything other then smartcharts folder inside js folder. Smartcharts is getting cached at line 164, and similarly for json that is also excluded